requirements/base.txt

```diff
@@ -41,5 +41,5 @@ pytz==2022.1
     #   djangorestframework
 sqlparse==0.4.2
     # via django
-stevedore==3.5.0
+stevedore==4.0.0
```

Comment:
Not any major incompatible change for now in the version [7.0.0](https://github.com/simplistix/testfixtures/blob/master/CHANGELOG.rst#700-6-jul-2022)